### ISSUE TITLE
Problem with recorded runs using folder

### ISSUE THREAD

**Issue by saramonzon** [BU-ISCIII/iskylims#200]
### Description of the bug

When you make all the wetlab process, the run is set to recorded properly, but then the crontab ignores the run because it says it can't find the folder although the folder is in the samba shared folder:
![image](https://github.com/BU-ISCIII/iskylims/assets/3480206/4cca231b-9b13-40e2-bd46-b50e219f0a79)

**Comment by saramonzon** [comment]
I'm really struggling with the crontab in docker, but now it seems to export an error which is more logic than doing nothing
I managed to run the crontab manually and here is the traceback:
```
2023-07-27 13:15:59,602 search_update_new_runs INFO     254 NextSeq_GEN_436 : Sequencer  stored on database
2023-07-27 13:15:59,603 looking_for_new_runs DEBUG    67 *************** ERROR *********************
2023-07-27 13:15:59,604 looking_for_new_runs DEBUG    68 RunningParameters matching query does not exist.
2023-07-27 13:15:59,604 looking_for_new_runs DEBUG    69 ################################
2023-07-27 13:15:59,604 looking_for_new_runs DEBUG    70 Traceback (most recent call last):
  File "/opt/iskylims/wetlab/cron.py", line 63, in looking_for_new_runs
    wetlab.utils.crontab_update_run.search_update_new_runs("crontab_request")
  File "/opt/iskylims/wetlab/utils/crontab_update_run.py", line 316, in search_update_new_runs
    run_folder = wetlab.models.RunningParameters.objects.get(
  File "/opt/iskylims/virtualenv/lib/python3.10/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/opt/iskylims/virtualenv/lib/python3.10/site-packages/django/db/models/query.py", line 435, in get
    raise self.model.DoesNotExist(
wetlab.models.RunningParameters.DoesNotExist: RunningParameters matching query does not exist.
```

**Comment by luissian** [comment]
When trying to fetch the remote folder, using the folder value stored in database, it fails because at this very moment RunningParameters do not save the value of the running parameters yet. So when executing this code fails 

```
 run_folder = wetlab.models.RunningParameters.objects.get(
    run_name_id__exact=run_process_obj
     ).get_run_folder()
```

There is not need to get the run folder because it is stored in new_run variable.  Then later when trying to save the sample sheet use the new_run variable instead
```
 try:
    wetlab.utils.crontab_process.copy_sample_sheet_to_remote_folder(
        conn, sample_sheet_path, new_run, experiment_name
     )
```
I think when trying to save sample sheet it will fails, because the samba folder is mounted read only mode, however the COPY_SAMPLE_SHEET_TO_REMOTE is set to true. I suggest change it to False

**Comment by luissian** [comment]
Applying these changes the issue was solved.
Fixed on commit 8f8ab52
